app.js

```diff
@@ -0,0 +1,124 @@
+'use-strict';
+//Global Variables
+
+
+
+
+// Create a constructor function that creates an object associated with each product
+
+function OddDuckProduct(name, imgPath){
+  this.name = name;
+  this.imgPath = imgPath;
+  this.imgTimesShown = 0;
+  this.imgTimesClicked = 0;
+}
+
+OddDuckProduct.prototype.render = function(n){
+
+  let imgContainer = document.getElementById(`image${n}`);
+  imgContainer.src = this.imgPath;
+};
+
+let allProducts = [
+  new OddDuckProduct('baby-sweep', 'img/baby-sweep.jpg'),
+  new OddDuckProduct('banana', 'img/banana.jpg'),
+  new OddDuckProduct('bathroom', 'img/bathroom.jpg'),
+  new OddDuckProduct('boots', 'img/boots.jpg'),
+  new OddDuckProduct('breakfast', 'img/breakfast.jpg'),
+  new OddDuckProduct('bubblegum', 'img/bubblegum.jpg'),
+  new OddDuckProduct('chair', 'img/chair.jpg'),
+  new OddDuckProduct('cthulhu', 'img/cthulhu.jpg'),
+  new OddDuckProduct('dog-duck', 'img/dog-duck.jpg'),
+  new OddDuckProduct('dragon', 'img/dragon.jpg'),
+  new OddDuckProduct('pen', 'img/pen.jpg'),
+  new OddDuckProduct('pet-sweep', 'img/pet-sweep.jpg'),
+  new OddDuckProduct('r2bag', 'img/r2bag.jpg'),
+  new OddDuckProduct('scissors', 'img/scissors.jpg'),
+  new OddDuckProduct('shark', 'img/shark.jpg'),
+  new OddDuckProduct('tauntaun', 'img/tauntaun.jpg'),
+  new OddDuckProduct('unicorn', 'img/unicorn.jpg'),
+  new OddDuckProduct('water-can', 'img/water-can.jpg'),
+  new OddDuckProduct('wine-glass', 'img/wine-glass.jpg'),
+];
+// Random number generator to select imgs from array
+
+function createRandomProducts(){
+  let randProduct = Math.floor(Math.random() * allProducts.length);
+  return allProducts[randProduct];
+}
+// Create render method for 1 image
```

Comment:
This makes two comments indicating code, while not having any. Add the described code or remove the comment.